sdk/eventgrid/azure-eventgrid/azure/eventgrid/_models.py

```diff
@@ -145,18 +164,39 @@ class EventGridEvent(InternalEventGridEvent, EventMixin):
     :param data_version: Required. The schema version of the data object.
         If not provided, will be stamped with an empty value.
     :type data_version: str
-    :param topic: The resource path of the event source. If not provided, Event Grid will stamp onto the event.
+    :keyword topic: The resource path of the event source. If not provided, Event Grid will stamp onto the event.
     :type topic: str
-    :ivar metadata_version: The schema version of the event metadata. If provided, must match Event Grid Schema exactly.
-        If not provided, EventGrid will stamp onto event.
-    :vartype metadata_version: str
-    :param data_version: The schema version of the data object. If not provided, will be stamped with an empty value.
+    :keyword metadata_version: The schema version of the event metadata. If provided, must match Event Grid Schema
+        exactly. If not provided, EventGrid will stamp onto event.
```

Comment:
not sure three space impacts doc rendering or not, but I usually just insert one space :P
```suggestion
    :keyword metadata_version: The schema version of the event metadata. If provided, must match Event Grid Schema
     exactly. If not provided, EventGrid will stamp onto event.
```